//package com.sincetimes.redwar.game.comnunicate.protocol;

message RseUpdateGameUnits
{
	optional int32 ret = 1;
}